src/app/views.py

```diff
@@ -78,7 +80,10 @@ def about(request):
     """ View for about
     returns about.html template
     """
-    context_dict={}
+    context_dict={
+                'spdx_version':spdx_version,
```

Comment:
Change this to `spdx_online_tools_version`

Reply:
spdx_version may be confused with with spdx specification version

Reply:
Agree with changing to online tools version